sandbox-backend/src/main/java/com/exadel/sandbox/team5/entity/Tag.java

```diff
@@ -17,4 +20,13 @@
 
     @Column(name = "tagName")
     private String name;
+
+    @JsonIgnore
```

Comment:
All JSON annotations should be used for DTOs

Reply:
got it.